### DISCUSSION TITLE
Lockscreen

### DISCUSSION THREAD

**Discussion by bayonatech** [linuxmint/discussions#245] [Ideas]
Hello, I would like to suggest a security improvement. When installing Linux Mint I set it so that the session starts automatically, when it starts I lock the screen and if I force it to shut down, when I return the screen comes back unlocked. In the case of a malicious person, if they want to have access to my data, they can use it. My suggestion would be if the session is locked if there is a power outage or a forced shutdown the screen goes back to a safe mode. If there is already a correction, please ignore it.

**Comment by SantiBurgos1089** [comment]
Greetings @bayonatech , you can check inside System settings - Screensaver - "Settings" tab, there is an option there in which you can set to lock the screen when put to sleep and how much time do you want to wait (the delay) until locking is applied (there is an option that says "immediatly")

I hope this works for you

![2023-12-12_08-46](https://github.com/linuxmint/discussions/assets/137421608/a1d9f7c4-318c-4a24-91c9-2e8aacb665f8)

**Comment by billyswong** [comment]
Automatic login doesn't work if your home directory is encrypted, for obvious reason. Also, a full disk encrypted computer is not vulnerable to your concern as one will need to enter password at boot just for booting into the login screen.

Therefore, your computer data is currently vulnerable to someone rebooting into a live ISO (e.g. the Mint installer ISO) then read data inside. Or pull out the drive from the computer then plug it into another computer to read the content. The feature you wish for is not going to save you.
